woocommerce/woocommerce-android
# [Woo POS] Make a WooLog.T for our project

### Closes
Closes: #12083

### Description
This PR introduces a new logging tag `Woo.T.POS` specifically for POS-related logs within the WooCommerce Android application. This enhancement aims to improve the clarity and ease of parsing logs related to POS functionality. 

### Steps to reproduce

**Test Log Outputs**:
   - Execute POS-related actions (e.g., create an order, add a product to the cart, handle order failures).
   - Inspect the log outputs to verify that the logs use the new `Woo.T.POS` tag instead of the general tags.

### Testing information
- **Devices Used**: Emulators 

### Release Notes
